cumulusci/core/keychain/encrypted_file_project_keychain.py

```diff
@@ -197,34 +257,69 @@ def _load_org_files(self, dirname: str, constructor=None):
                     constructor(config) if constructor else config
                 )
 
-    def _set_org(self, org_config, global_org):
+    def _set_org(self, org_config, global_org, save=True):
+        if not global_org and not self.project_local_dir:
+            return
```

Comment:
When does this happen? Should it raise an exception?

Reply:
Hmm.... not sure if this is even possible. You wouldn't be able to operate on a local org without a CumulusCI project being present, which would necessitate there being a project_local_dir 🤔